app/analysis/index/controller.js

```diff
@@ -35,228 +35,202 @@ const deserializeMoment = (momentString) =>
 
 export const AnalysisQueryParams = new QueryParams({
   customer: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   project: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   task: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   user: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   reviewer: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   billingType: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   costCenter: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
   },
   fromDate: {
-    defaultValue: null,
+    defaultValue: undefined,
     replace: true,
     refresh: true,
     serialize: serializeMoment,
     deserialize: deserializeMoment,
   },
   toDate: {
-    defaultValue: null,
+    defaultValue: undefined,
```

Comment:
`null` as default value leads to a `change` in the this.queryParamsState Object. And the change indicated is `null` -> `undefined`.  So I assume the defaultValue for query params if not defined should be `undefined` instead of `null`.